.github/workflows/pr-title-emoji.yml

```
@@ -0,0 +1,115 @@
+    types: [opened, edited, synchronize]
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+
+jobs:
+  format-pr-title:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.ref != 'main' && github.event.pull_request.head.ref != 'develop'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)